nerfstudio/data/datamanagers/full_images_datamanager.py

```diff
@@ -420,6 +422,9 @@ def _undistort_image(
         # crop the image and update the intrinsics accordingly
         x, y, w, h = roi
         image = image[y : y + h, x : x + w]
+        # update the principal point based on our cropped region of interest (ROI)
+        newK[0, 1] -= x
+        newK[0, 2] -= y
```

Comment:
actually if we're adjusting cx and cy shouldn't this be `newK[0, 2]` and `newK[1, 2]`?

Reply:
yes this is right! ahh sorry i mistakenly flipped in the indices